packages/scaffold-config/src/frameworks.ts

```diff
@@ -0,0 +1,579 @@
+import dedent from 'dedent'
+import type { Bundler } from './types'
+import {
+  CODE_GEN_FRAMEWORKS,
+  FRONTEND_FRAMEWORK_CATEGORIES,
+  CYPRESS_REACT_LATEST,
+  CYPRESS_VUE_2,
+  CYPRESS_VUE_3,
+  HTML_WEBPACK_PLUGIN_4,
+  STORYBOOK_REACT,
+  STORYBOOK_VUE,
+  WEBPACK_DEV_SERVER_4,
+  BUNDLER_WEBPACK_4,
+  CYPRESS_WEBPACK,
+} from './constants'
+import { BUNDLER_VITE, BUNDLER_WEBPACK_5, HTML_WEBPACK_PLUGIN_5 } from '.'
+
+const isWebpack = (bundler: Bundler) => ['webpack4', 'webpack5'].includes(bundler)
+
+export const FRONTEND_FRAMEWORKS = [
+  {
```

Comment:
Could each of those be in its own file?
I could see myself scrolling for hours to find the framework I am looking to debug